DigitalLearningSolutions.Web/Views/TrackingSystem/CourseSetup/_CentreCourseCard.cshtml

```diff
@@ -8,7 +8,62 @@
         @Model.CourseName
       </span>
     </summary>
+
     <div class="nhsuk-details__text">
+      <partial name="SearchablePage/_FilterableTags" model="@Model.Tags" />
+
+      <dl class="nhsuk-summary-list">
+        <div class="nhsuk-summary-list__row">
+          <dt class="nhsuk-summary-list__key">
+            Category
+          </dt>
+          <dd class="nhsuk-summary-list__value">
+            @Model.CategoryName
+          </dd>
+        </div>
+
+        <div class="nhsuk-summary-list__row">
+          <dt class="nhsuk-summary-list__key">
+            Learning minutes
+          </dt>
+          <dd class="nhsuk-summary-list__value">
+            @Model.LearningMinutes
+          </dd>
+        </div>
+
+        <div class="nhsuk-summary-list__row">
+          <dt class="nhsuk-summary-list__key">
+            Total delegates
+          </dt>
+          <dd class="nhsuk-summary-list__value">
+            @Model.DelegateCount
+          </dd>
+        </div>
+
+        <div class="nhsuk-summary-list__row">
+          <dt class="nhsuk-summary-list__key">
+            In progress
+          </dt>
+          <dd class="nhsuk-summary-list__value">
+            @Model.InProgressCount
+          </dd>
+        </div>
+
+      </dl>
+
+      <div class="nhsuk-grid-row">
+        <div class="nhsuk-grid-column-full">
+          <p class="nhsuk-u-margin-bottom-0">Want to share the course?</p>
+          <a href="@Model.GenerateEmailHref">Generate email</a>
+        </div>
+
+      </div>
+      <div class="nhsuk-grid-row">
+        <div class="nhsuk-grid-column-full">
+          <a href="#" class="js-only-inline copy-course-link nhsuk-link--no-visited-state" id="copy-course-@Model.CustomisationId">Copy course link</a>

```

Comment:
Just wanted to double check if you've checked that this is the most appropriate HTML for this functionality (for clarity - I haven't looked into it at all)